### ISSUE TITLE
Python>=3.8 requires pygame>=2.0, but pgzero requires pygame<2.0

### ISSUE THREAD

**Issue by lgautier** [lordmauve/pgzero#225]
Python 3.8 is the default Python3 on ubuntu 20.04.
Is there a list of issues preventing pgzero from working with pygame2 ?

**Comment by lgautier** [comment]
When just removing the requirement for pygame < 2.0 in `setup.py` for `pgzero 1.3.dev0` I get the following when running tests:

```
=========================== short test summary info ============================
FAILED test/test_screen.py::ScreenTest::test_circle - AssertionError: Images ...
FAILED test/test_screen.py::ScreenTest::test_filled_circle - AssertionError: ...
FAILED test/test_screen.py::ScreenTest::test_line - AssertionError: Images di...
FAILED test/test_screen.py::ScreenTest::test_polygon - AssertionError: Images...
FAILED test/test_screen.py::ScreenTest::test_wrapped_gradient_text - Assertio...
FAILED test/test_sound_formats.py::SoundFormatsTest::test_load_8k16bitpcm
FAILED test/test_sound_formats.py::SoundFormatsTest::test_load_8k8bitpcm
FAILED test/test_sound_formats.py::SoundFormatsTest::test_load_8kadpcm
============= 8 failed, 157 passed, 4 skipped, 2 warnings in 2.01s =============
```

**Comment by lgautier** [comment]
The PR #207 seems to suggest a similar fix.
Are there plans to merge it ?

**Comment by dlebauer** [comment]
@lgautier #207 has been merged, so this can be closed

**Comment by lordmauve** [comment]
I have enquired with the Pygame maintainers about getting wheels produced for Python 3.8/3.9 for Pygame 1.9.6, because that is the only issue here.

They are focusing on Pygame 2 but there's no technical reason they cannot be generated and uploaded.

**Comment by lordmauve** [comment]
I've released a new version 1.2.1 which pins the Pygame dependency to 2.0. I also investigated and fixed the sound format and drawing tests (on the 1.2release branch).
